ibis_vega_transform/vegaexpr.py

```diff
@@ -11,11 +11,61 @@
 
 import ibis
 import ibis.expr.types as it
-import pytz
 from mypy_extensions import TypedDict
 from typing_extensions import Literal
 
 from altair_transform.utils import evaljs
+import altair_transform.utils._evaljs
+
+# Monkey patch altair_transform so that boolean operators  work on ibis expression
+
+
+def not_operator(value):
+    if isinstance(value, ibis.expr.types.ValueExpr):
+        return ~value
+    return not value
+
+
+def and_operator(l, r):
+    if isinstance(l, ibis.expr.types.ValueExpr):
+        if isinstance(r, ibis.expr.types.ValueExpr):
+            return l & r
+        return r and l
+    return l and r
+
+
+def or_operator(l, r):
+    if isinstance(l, ibis.expr.types.ValueExpr):
+        if isinstance(r, ibis.expr.types.ValueExpr):
+            return l & r
```

Comment:
Is this a typo? Seems like it should be `l | r`

Reply:
Yep, thank you!